packages/common/src/intl/IntlProvider.tsx

```diff
@@ -12,7 +12,7 @@ export const IntlProvider: React.FC = ({ children }) => {
     const minuteInMilliseconds = 60 * 1000;
     const expirationTime =
       process.env['NODE_ENV'] === 'development'
-        ? 5 * minuteInMilliseconds
+        ? 0
```

Comment:
I got fooled a couple times already by localisations not updating 😁  I think this works when you refresh without having to clear local storage

Reply:
yah, caching 🙄